pkg/mods/flag/flag.go

```diff
@@ -30,7 +30,9 @@ func call(fm *eval.Frame, fn *eval.Closure, argsVal vals.List) error {
 	fs := newFlagSet("")
 	for i, name := range fn.OptNames {
 		value := fn.OptDefaults[i]
-		addFlag(fs, name, value, "")
+		if err := addFlag(fs, name, value, ""); err != nil {
```

Comment:
This pattern is popular in certain Go projects as it saves one line of code, but IMO `if` with an init clause should only be used when the init clause is only used to calculate something that's only used in the condition or the if body. In this case `addFlag` has side effects that's relevant outside the `if` block.